package.json

```diff
@@ -5,7 +5,11 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+<<<<<<< Updated upstream
     "start": "nodemon server.js",
+=======
+    "start": "node server.js",
```

Comment:
Please have the original, "start":nodemon server.js

Reply:
I changed it